safegraph/README.md

```diff
@@ -1,24 +1,24 @@
-# Safegraph Mobility Data
+# Patterns Dataset in Safegraph Mobility Data
 
-We import raw mobility data from Safegraph, calculate some statistics upon it,
-and aggregate the data from the Census Block Group to County and State levels.
-For detailed information see the files `DETAILS.md` contained
-in this directory.
+We import raw mobility data from Safegraph, calculate some 
+statistics upon it, and aggregate the data from the Zip Code level to County,
+HRR, MSA and State levels. For detailed information see the files `DETAILS.md` 
+contained in this directory.
```

Comment:
Apologies; this change is also bogus; Patterns comes to us as Zip but the vanilla Mobility is CBGs.

Reply:
My bad, I was sloppy doing the copy paste. I saw quidel was pretty much a straight replacement and assumed sg would be as well.